llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -407,6 +407,54 @@ def SYNTACORE_SCR7 : RISCVProcessorModel<"syntacore-scr7",
                                                FeatureStdExtZkn],
                                               [TuneNoDefaultUnroll, FeaturePostRAScheduler]>;
 
+def TENSTORRENT_ASCALON_D8 : RISCVProcessorModel<"tt-ascalon-d8",
```

Comment:
Is it OK to use `tt` here? Shoule it be the full name `tenstorrent`?

Reply:
We'd prefer `tt` as it would be shorter :smile:

Reply:
Yeah I agree it is shorter. I don't know if this is a de-facto policy here, we seem to use full name for all CPUs. For example, we don't use `sf` for SiFive, etc.

Reply:
Just from a quick Google search, Tenstorrent frequently uses TT in branding of their products, not just CPUs.

Reply:
With my company hat on I can say that we would prefer tt-ascalon-d8, we use lowercase abbreviation 'tt' in other places. For example our lower level AI stack is referred to as 'tt-metal' (as opposed to 'tenstorrent-tt-metal' or 'tenstorrent-metal')

Reply:
Make sense to me then.